blog.html

```diff
@@ -33,7 +33,7 @@ <h1 class="title page-title">The Teach LA Blog</h1>
         {% endfor %}
     {% endif %}
     {% endcomment %}
-  <div class="post-preview">
+  <!-- <div class="post-preview">
```

Comment:
Go ahead and delete this commented out section. The blog-preview-card should output all of the same information, so we no longer need this commented section.

Reply:
Oops I meant to delete this, my bad! I'll do that now!